stories/Popover.stories.js

```diff
@@ -12,7 +12,7 @@ import { withArtboard } from '@helpscout/artboard'
 import { action } from '@storybook/addon-actions'
 
 const stories = storiesOf('Popover', module)
-stories.addDecorator(withArtboard())
+stories.addDecorator(withArtboard({ width: 300, height: 100 }))
```

Comment:
oh I didn't know you could set a custom width

can you do a fullscreen artboard?

Reply:
@plbabin I've never tried! Maybe? haha 😂